### PR TITLE
48 create first test for react

### LINKED ISSUE #48: Create first test for react
## Describe the feature
Create a test for React with documentation to know how to create a test and how to run them

## TO DO
- Create a test folder for React
- Create a first test file
- Create a how-to-use file to document test

## Extra links (Optional)
Try using this framework: [playwrigth](https://playwright.dev/)

## Screenshots (Optional)
_Add any screenshot_

### PR DESCRIPTION
**Pull request by osvegn**
# New feature
- Use playwright to create tests for web part

### CODE REVIEW

File: Web/src/HomePage.js [hunk 1]
```diff
@@ -243,14 +243,15 @@ export default function HomePage(props) {
                 alignItems: "center",
                 backgroundColor: "lightgrey",
                 width: "300px",
-                height: "150px",
+                height: "400px",
+                // overflow: "scroll",
```
Comment (31Nathan) [inline Web/src/HomePage.js]: Remove it if it's unused

File: Web/playwright.config.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    /* Test against mobile viewports. */
+    // {
+    //   name: 'Mobile Chrome',
+    //   use: { ...devices['Pixel 5'] },
+    // },
+    // {
+    //   name: 'Mobile Safari',
+    //   use: { ...devices['iPhone 12'] },
+    // },
+
```
Comment (31Nathan) [inline Web/playwright.config.js]: Why commented ?

**Review by 31Nathan — CHANGES_REQUESTED** [review]
Some things

**Review by STom6 — DISMISSED** [review]
lgtm

**Review by JohanCDev — APPROVED** [review]
Ntr

**Review by 31Nathan — APPROVED** [review]
LGTM

**Review by STom6 — APPROVED** [review]
lgtm

**Pull request merged**